README.md

```diff
@@ -194,6 +200,90 @@ Detailed logs and results are available on Neoload Web. To get the url of the cu
 neoload logs-url                        # The URL to the test in Neoload Web
 ```
 
+### The test-results vs. report subcommands
+
+The 'test-results' subcommand is intended for direct operational queries against high-level API data.
+
+The 'report' subcommand is intended to simplify not only common data exporting needs, but also provide
+ templating capabilities over a standard, correlated data model. In contrast to the test-results
+ subcommand, 'report' can be used to generate as well as transform test result data.
+
+### Exporting Transaction CSV data
+```
+Usage: neoload report [OPTIONS] [[single|trends]] [NAME]
```

Comment:
The "Usage" is not up to date.
The example command does not work because "single" is no more an argument.

Reply:
0ac6fe6 Updated README to remove unnecessary options when defaults are sufficient (single cur)